src/app/tweets/tweets.component.html

```diff
@@ -45,35 +50,50 @@ <h4 class="header-title m-t-0">All Tweets</h4>
       <p class="text-muted m-b-25 font-13">
         All tweets from @realDonaldTrump
       </p>
-      <table class="table mb-0 table-responsive">
-        <thead>
-          <tr>
-            <th>Date</th>
-            <th>Time</th>
-            <th>Phrases</th>
-            <th>Sentiment</th>
-            <th>Tweet</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr *ngFor="let tweet of tweets$ | async | slice:0:100 ">
-            <td>{{tweet.ts | date: 'fullDate'}}</td>
-            <td>{{tweet.ts | date: 'shortTime'}}</td>
-            <td>
-              <div *ngFor="let phrase of tweet.phrases | slice:0:3">
-                <ins class="text-default">
-                  {{phrase}}</ins>&nbsp; |
-              </div>
-            </td>
-            <td>
-              <img src="assets/images/logo_postive.png" width="80" height="80" alt="logo" *ngIf="tweet.sentiment > 0.66">
-              <img src="assets/images/logo.png" width="80" height="80" alt="logo" *ngIf="tweet.sentiment <= 0.66 && tweet.sentiment >= 0.33">
-              <img src="assets/images/logo_negative.png" width="80" height="80" alt="logo" *ngIf="tweet.sentiment < 0.33">
-            </td>
-            <td>{{tweet.text}}</td>
-          </tr>
-        </tbody>
-      </table>
+
+      <div class="list-group">
+        <a href="#"
+           *ngFor="let tweet of tweets$ | async | slice:0:100"
+           class="list-group-item list-group-item-action flex-column align-items-start">
+          <div class="d-flex w-100 justify-content-between">
+            <h5 class="mb-2">
+
+              <img src="assets/images/logo_postive.png"
+                   width="80"
+                   height="80"
+                   alt="logo"
+                   *ngIf="tweet.sentiment > 0.66">
+              <img src="assets/images/logo.png"
+                   width="80"
+                   height="80"
+                   alt="logo"
+                   *ngIf="tweet.sentiment <= 0.66 && tweet.sentiment >= 0.33">
+              <img src="assets/images/logo_negative.png"
+                   width="80"
+                   height="80"
+                   alt="logo"
+                   *ngIf="tweet.sentiment < 0.33">
+
+            </h5>
+            <small>{{ timeAgo(tweet.ts) }}</small>
+          </div>
+
+          <p class="mb-2">{{tweet.text}}</p>
```

Comment:
Let's try converting this to an anchor, so when you click the tweet it takes you to the actual tweet (set target="_blank" so it opens in a new page)

To get the tweet itself it's "twitter.com/statuses" + id_str